.travis.yml

```diff
@@ -34,6 +34,7 @@ env:
     - KONG_TEST_PG_USER=postgres
     - JOBS=2
     - DOCKER_MACHINE_ARM64_NAME=travis-ci-kong-${TRAVIS_JOB_ID}
+    - OPENRESTY_PATCHES_BRANCH=chore/cosocket_mtls
```

Comment:
(will be reverted once https://github.com/Kong/openresty-patches/pull/57 is merged)